src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/target.py

```diff
@@ -95,6 +95,14 @@ def _target_is_account_id(self):
         ).get('Account')
         self._create_response_object([responses])
 
+    def _target_is_tags(self):
+        responses = self.organizations.get_account_ids_for_tags(self.path)
```

Comment:
Could you rename `responses` to `account_ids`?

Reply:
In the entire class the responses from boto3 calls are called responses. I think being consequent is better. The entire class could be refactored but I dont want to add functionality and refactor at the same time unless a refactoring is needed in order to add the functionality in a decent way.

Reply:
I would propose getting the boto3 calls renamed in that case too, but that would be a separate PR.

In this case we are not calling a boto3 function though.
You are invoking this [function (.../shared/python/organizations.py#L137-R153)](https://github.com/awslabs/aws-deployment-framework/pull/163/files#diff-f47c805f0dd7e769092ac78c5b434ed4R137-R153), or am I mistaken?